### DISCUSSION TITLE
Argo CD

### DISCUSSION THREAD

**Discussion by hairmare** [valkey-io/valkey#340] [Show and tell]
Hi ValKey community

I recently started drive-by maintaining the switch to ValKey for the Argo CD project, see here https://github.com/argoproj/argo-cd/issues/17892 for my efforts

let's go!

**Comment by gtudan** [comment]
@carrodher mentioned in the #65 that Bitnami might provide a chart once the docker images are available. Any news since then?

**Reply by carrodher** [reply]
We're actively working on the Bitnami Valkey container images and Helm chart, I hope it will be ready throughout May 🤞

**Reply by madolson** [reply]
@carrodher If you need any help, don't hesitate to reach out. We're glad you're considering us!

**Reply by carrodher** [reply]
We just released the container images, see the images in DockerHub
- https://hub.docker.com/r/bitnami/valkey
- https://hub.docker.com/r/bitnami/valkey-sentinel

and the source code in GitHub
- https://github.com/bitnami/containers/tree/main/bitnami/valkey
- https://github.com/bitnami/containers/tree/main/bitnami/valkey-sentinel

Any feedback is more than welcome, you can report any issue or suggestion at https://github.com/bitnami/containers/issues/new/choose

Now it's time for the Helm chart! 🚀

**Reply by carrodher** [reply]
At https://github.com/bitnami/charts/pull/25643 you can see the PR implementing the Valkey Helm chart

**Comment by carrodher** [comment]
I'm excited to share that we've just released the Bitnami Valkey Helm chart, now available on Docker Hub at https://hub.docker.com/r/bitnamicharts/valkey, simplifying the deployment of Valkey in Kubernetes.

Built upon the same principles as our other Bitnami charts, Valkey follows the latest best practices and security industry standards. Learn more about our commitment to security [here](https://blog.bitnami.com/2024/04/bitnami-helm-charts-are-now-more-secure.html).

We welcome your feedback and contributions on GitHub. Whether it's reporting issues, suggesting improvements, or contributing code, your input is invaluable so we can, together, continue to enhance the product.
